IBAnimatable/AnimatableBarButtonItem.swift

```diff
@@ -28,22 +28,22 @@ import UIKit
 //  }
   
   // MARK: - Animatable
-  public var animationType: AnimationType = .none
+  open var animationType: AnimationType = .none
   @IBInspectable var _animationType: String? {
     didSet {
       animationType = AnimationType(string: _animationType)
     }
   }
-  @IBInspectable public var autoRun: Bool = true
-  @IBInspectable public var duration: Double = Double.nan
-  @IBInspectable public var delay: Double = Double.nan
-  @IBInspectable public var damping: CGFloat = CGFloat.nan
-  @IBInspectable public var velocity: CGFloat = CGFloat.nan
-  @IBInspectable public var force: CGFloat = CGFloat.nan
-  @IBInspectable public var repeatCount: Float = Float.nan
+  @IBInspectable open var autoRun: Bool = true
```

Comment:
Should it be open instead of public? Will it be a reason to override this variables?